modules/product/product.service.js

```diff
@@ -15,13 +15,13 @@ class ProductsService {
   }
 
   getModelsByCategory(id) {
-    return Model.find({ category: id });
+    return Product.find({ category: id });
```

Comment:
Wait, should it be `Product` or `Model`

Reply:
I couldn't commit cause of this error, someone imported Product from './product.model', but used Model, that was not defined